packages/wdio-utils/src/driver/utils.ts

```diff
@@ -201,12 +201,30 @@ export function getCacheDir (options: Pick<Options.WebDriver, 'cacheDir'>, caps:
     return driverOptions.cacheDir || options.cacheDir || os.tmpdir()
 }
 
+export function getMajorVersionFromString(fullVersion:string) {
+    let prefix
+    if (fullVersion) {
+        prefix = fullVersion.match(/^[+-]?([0-9]+)/)
+    }
+    return prefix && prefix.length > 0 ? prefix[0] : ''
+}
+
+export async function getKnownBuild (build: string) {
+    log.warn(`Chromedriver v${build} don't exist, trying to find known good version...`)
+    const knownGoodVersions: any = await got('https://googlechromelabs.github.io/chrome-for-testing/known-good-versions.json').json()
```

Comment:
This still uses a download url instead of the Puppeteer package.

Reply:
Oh yes! The func needn't at all 'resolveBuildId' can parse major version and return good known. I've added changes